apps/judicial-system/api/src/app/modules/case/models/pendingSignature.model.ts

```diff
@@ -0,0 +1,12 @@
+import { Field, ObjectType } from '@nestjs/graphql'
+
+import { PendingSignature as TPendingSignature } from '@island.is/judicial-system/types'
+
+@ObjectType()
+export class PendingSignature implements TPendingSignature {
+  @Field()
+  controlCode: string
```

Comment:
I'm guessing this is the ID for the type. Then this needs to be marked as:
```typescript
import { Field, ID } from '@nestjs/graphql'
...
@Field((_1) => ID)
```
for better caching in the front-end.

Reply:
^ More of an awareness for all your GraphQL types

Reply:
Í þessu tilviki er ekki um raunverulegan object að ræða og ekkert id. Þetta er svar sem við fáum frá undirskriftarþjónustu sem notar rafræn skilríki á sími. Þetta svæði (controlCode) er fjögurra tölustafa kóðinn sem birtist á símanum þínum þegar þú undirritar.